cfgov/v1/models/molecules.py

```diff
@@ -15,14 +11,14 @@ def isRequired(field_name):
     return [str(field_name) + ' is required.']
 
 
-class HalfWidthLinkBlob(blocks.StructBlock):
+class LinkBlob(blocks.StructBlock):
     heading = blocks.CharBlock(max_length=100, required=True)
     body = blocks.RichTextBlock(blank=True)
     links = blocks.ListBlock(atoms.Hyperlink(), required=False)
 
     class Meta:
         icon = 'link'
-        template = '_includes/molecules/half-width-link-blob.html'
+        template = '_includes/molecules/link-blob.html'
```

Comment:
If you're deleting the `link-blob` template, this will need to be reverted right?